core/plugins/org.fusesource.ide.camel.model.service.v2_17_3/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Fuse Foundation UI Plugin
```

Comment:
I think we can improve the bundle name.
Proposal: Fuse Camel Model Service 2.17.3